### ISSUE TITLE
'Warning: bundle "" already exists; skipping.' when using mixer init --all

### ISSUE THREAD

**Issue by cmarcelo** [clearlinux/mixer-tools#126]
The full output:

```
$ mixer init --clear-version 20600 --mix-version 10 --all
Creating new builder.conf configuration file...
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Warning: bundle "" already exists; skipping.
Adding bundle "R-basic"
Adding bundle "R-extras"
Adding bundle "amateur-radio"
Adding bundle "application-server"
Adding bundle "big-data-basic"
Adding bundle "bootloader"
Adding bundle "bootloader-extras"
Adding bundle "c-basic"
Adding bundle "cloud-control"
Adding bundle "cloud-native-basic"
Adding bundle "clr-devops"
Adding bundle "containers-basic"
Adding bundle "containers-basic-dev"
Adding bundle "containers-virt"
Adding bundle "containers-virt-dev"
Adding bundle "cryptography"
Adding bundle "database-basic"
Adding bundle "database-basic-dev"
Adding bundle "desktop"
Adding bundle "desktop-apps"
Adding bundle "desktop-apps-extras"
Adding bundle "desktop-assets"
Adding bundle "desktop-autostart"
Adding bundle "desktop-dev"
Adding bundle "desktop-gnomelibs"
Adding bundle "desktop-locales"
Adding bundle "dev-utils"
Adding bundle "dev-utils-dev"
Adding bundle "dhcp-server"
Adding bundle "editors"
Adding bundle "editors-dev"
Adding bundle "games"
Adding bundle "games-dev"
Adding bundle "go-basic"
Adding bundle "go-basic-dev"
Adding bundle "haskell-basic"
Adding bundle "hpc-utils"
Adding bundle "java-basic"
Adding bundle "kernel-aws"
Adding bundle "kernel-container"
Adding bundle "kernel-gce"
Adding bundle "kernel-hyperv"
Adding bundle "kernel-hyperv-lts"
Adding bundle "kernel-hyperv-mini"
Adding bundle "kernel-kvm"
Adding bundle "kernel-lts"
Adding bundle "kernel-native"
Adding bundle "koji"
Adding bundle "kvm-host"
Adding bundle "letsencrypt-client"
Adding bundle "libX11client"
Adding bundle "machine-learning-basic"
Adding bundle "machine-learning-web-ui"
Adding bundle "mail-utils"
Adding bundle "mail-utils-dev"
Adding bundle "mixer"
Adding bundle "network-basic"
Adding bundle "network-basic-dev"
Adding bundle "nodejs-basic"
Adding bundle "openssh-server"
Adding bundle "os-clear-containers"
Adding bundle "os-cloudguest"
Adding bundle "os-cloudguest-aws"
Adding bundle "os-cloudguest-azure"
Adding bundle "os-cloudguest-gce"
Adding bundle "os-cloudguest-vmware"
Adding bundle "os-clr-on-clr"
Adding bundle "os-clr-on-clr-dev"
Adding bundle "os-core"
Adding bundle "os-core-dev"
Adding bundle "os-core-update"
Adding bundle "os-core-update-dev"
Adding bundle "os-installer"
Adding bundle "os-testsuite"
Adding bundle "os-testsuite-phoronix"
Adding bundle "os-utils-gui"
Adding bundle "os-utils-gui-dev"
Adding bundle "performance-tools"
Adding bundle "perl-basic"
Adding bundle "perl-basic-dev"
Adding bundle "perl-extras"
Adding bundle "php-basic"
Adding bundle "pnp-tools-basic"
Adding bundle "productivity"
Adding bundle "pxe-server"
Adding bundle "python-basic"
Adding bundle "python-basic-dev"
Adding bundle "python-extras"
Adding bundle "python3-basic"
Adding bundle "qt-basic"
Adding bundle "ruby-basic"
Adding bundle "rust-basic"
Adding bundle "shells"
Adding bundle "storage-cluster"
Adding bundle "storage-utils"
Adding bundle "storage-utils-dev"
Adding bundle "stream-server"
Adding bundle "sysadmin-basic"
Adding bundle "sysadmin-basic-dev"
Adding bundle "sysadmin-hostmgmt"
Adding bundle "sysadmin-remote"
Adding bundle "tcl-basic"
Adding bundle "telemetrics"
Adding bundle "user-basic"
Adding bundle "user-basic-dev"
Adding bundle "vnc-server"
Adding bundle "web-server-basic"
Adding bundle "xfce4-desktop"
Initialized empty Git repository in /home/c/src/mix3/mix-bundles/.git/
[master (root-commit) 67aabca] Initial mix version 10 from upstream version 20600
 108 files changed, 8755 insertions(+)
 create mode 100644 R-basic
 create mode 100644 R-extras
 create mode 100644 amateur-radio
 create mode 100644 application-server
 create mode 100644 big-data-basic
 create mode 100644 bootloader
 create mode 100644 bootloader-extras
 create mode 100644 c-basic
 create mode 100644 cloud-control
 create mode 100644 cloud-native-basic
 create mode 100644 clr-devops
 create mode 100644 containers-basic
 create mode 100644 containers-basic-dev
 create mode 100644 containers-virt
 create mode 100644 containers-virt-dev
 create mode 100644 cryptography
 create mode 100644 database-basic
 create mode 100644 database-basic-dev
 create mode 100644 desktop
 create mode 100644 desktop-apps
 create mode 100644 desktop-apps-extras
 create mode 100644 desktop-assets
 create mode 100644 desktop-autostart
 create mode 100644 desktop-dev
 create mode 100644 desktop-gnomelibs
 create mode 100644 desktop-locales
 create mode 100644 dev-utils
 create mode 100644 dev-utils-dev
 create mode 100644 dhcp-server
 create mode 100644 editors
 create mode 100644 editors-dev
 create mode 100644 games
 create mode 100644 games-dev
 create mode 100644 go-basic
 create mode 100644 go-basic-dev
 create mode 100644 haskell-basic
 create mode 100644 hpc-utils
 create mode 100644 java-basic
 create mode 100644 kernel-aws
 create mode 100644 kernel-container
 create mode 100644 kernel-gce
 create mode 100644 kernel-hyperv
 create mode 100644 kernel-hyperv-lts
 create mode 100644 kernel-hyperv-mini
 create mode 100644 kernel-kvm
 create mode 100644 kernel-lts
 create mode 100644 kernel-native
 create mode 100644 koji
 create mode 100644 kvm-host
 create mode 100644 letsencrypt-client
 create mode 100644 libX11client
 create mode 100644 machine-learning-basic
 create mode 100644 machine-learning-web-ui
 create mode 100644 mail-utils
 create mode 100644 mail-utils-dev
 create mode 100644 mixer
 create mode 100644 network-basic
 create mode 100644 network-basic-dev
 create mode 100644 nodejs-basic
 create mode 100644 openssh-server
 create mode 100644 os-clear-containers
 create mode 100644 os-cloudguest
 create mode 100644 os-cloudguest-aws
 create mode 100644 os-cloudguest-azure
 create mode 100644 os-cloudguest-gce
 create mode 100644 os-cloudguest-vmware
 create mode 100644 os-clr-on-clr
 create mode 100644 os-clr-on-clr-dev
 create mode 100644 os-core
 create mode 100644 os-core-dev
 create mode 100644 os-core-update
 create mode 100644 os-core-update-dev
 create mode 100644 os-installer
 create mode 100644 os-testsuite
 create mode 100644 os-testsuite-phoronix
 create mode 100644 os-utils-gui
 create mode 100644 os-utils-gui-dev
 create mode 100644 performance-tools
 create mode 100644 perl-basic
 create mode 100644 perl-basic-dev
 create mode 100644 perl-extras
 create mode 100644 php-basic
 create mode 100644 pnp-tools-basic
 create mode 100644 productivity
 create mode 100644 pxe-server
 create mode 100644 python-basic
 create mode 100644 python-basic-dev
 create mode 100644 python-extras
 create mode 100644 python3-basic
 create mode 100644 qt-basic
 create mode 100644 ruby-basic
 create mode 100644 rust-basic
 create mode 100644 shells
 create mode 100644 storage-cluster
 create mode 100644 storage-utils
 create mode 100644 storage-utils-dev
 create mode 100644 stream-server
 create mode 100644 sysadmin-basic
 create mode 100644 sysadmin-basic-dev
 create mode 100644 sysadmin-hostmgmt
 create mode 100644 sysadmin-remote
 create mode 100644 tcl-basic
 create mode 100644 telemetrics
 create mode 100644 user-basic
 create mode 100644 user-basic-dev
 create mode 100644 vnc-server
 create mode 100644 web-server-basic
 create mode 100644 xfce4-desktop
```

**Comment by kevincwells** [comment]
Huh. Let me look into this.

**Comment by kevincwells** [comment]
Fixed in #129. All of that code goes away in my bundle refactor, but it's an easy fix and a clean merge.
